packages/web/src/components/organisms/TriggersFormItems/ActionFormItems.tsx

```diff
@@ -16,12 +16,19 @@ const ActionFormItems = () => {
 
   const [switcherValue, setSwitcherValue] = useState('label');
 
-  const nameSelector = Form.useFormInstance().getFieldValue('testNameSelector');
+  const form = Form.useFormInstance();
+
+  const actionValue = Form.useWatch('action', form);
+  const nameSelector = form.getFieldValue('testNameSelector');
```

Comment:
Optionally `useWatch` could be used there too (to use subscription instead of one time value), but not important 🙂